products/nix.md

```diff
@@ -75,7 +80,7 @@ releases:
     latest: "2.1.3"
     eol: true
     latestReleaseDate: 2018-10-02
-    releaseDate: 2018-09-03
+    releaseDate: 2018-09-02
```

Comment:
@BiNZGi unfortunately there is a risk that those dates are reverted next time the `latest.py` script run (if it is not the same as the git tag date on https://github.com/NixOS/nix/releases/tag/).